### PR TITLE
Typo "Microsoft Teams Room"→"Microsoft Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/hardware-requirements-for-the-teams-app 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/hardware-requirements-for-the-teams-app.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/hardware-requirements-for-the-teams-app.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ All of the requirements in the following sections apply to both the Microsoft Te
...
 |Video  |    Compatible webcam     |
 |Voice    |  Compatible microphone and speakers, headset with microphone, or equivalent device       |
-|Video calls and meetings | <ul><li>Requires two-core processor. For higher video/screen share resolution and frame rate, a four-core processor or better is recommended. </li><li>Joining a meeting using proximity detection in a Microsoft Teams Room requires Bluetooth LE. Bluetooth LE requires Bluetooth to be enabled on the client device.</li></ul>
+|Video calls and meetings | <ul><li>Requires two-core processor. For higher video/screen share resolution and frame rate, a four-core processor or better is recommended. </li><li>Joining a meeting using proximity detection in a Microsoft Teams Rooms requires Bluetooth LE. Bluetooth LE requires Bluetooth to be enabled on the client device.</li></ul>
```
Comment (JohanFreelancer9) [inline Teams/hardware-requirements-for-the-teams-app.md:58]:
```suggestion
|Video calls and meetings | <ul><li>Requires two-core processor. For higher video/screen share resolution and frame rate, a four-core processor or better is recommended. </li><li>Joining a meeting using proximity detection in Microsoft Teams Rooms requires Bluetooth LE. Bluetooth LE requires Bluetooth to be enabled on the client device.</li></ul>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested change. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@tonysmit can you review?

**Pull request merged**